.github/workflows/code.yml

```
@@ -72,7 +72,16 @@ jobs:
         run: cargo ${{ matrix.plan.mode.cargoCommand }} ${{ matrix.plan.mode.cargoArgs }}
 
   e2e:
-    name: End-to-end tests
+    strategy:
+      matrix:
+        qualifier: ["base", "try-runtime"]
+        include:
+          - qualifier: "base"
+            extend-build-args: ""
+          - qualifier: "try-runtime"
+            extend-build-args: "--features try-runtime"
```

```suggestion
      matrix:
        test:
          - qualifier: "base"
            extend-build-args: ""
          - qualifier: "try-runtime"
            extend-build-args: "--features try-runtime"
```

Better use the matrix like this, it is easier to understand.

---

```suggestion
      matrix:
        test:
          - suite: "base"
            build-args: ""
          - suite: "try-runtime"
            build-args: "--features try-runtime"
```

And this is my further suggestion to tweak the key names

---

https://github.com/humanode-network/humanode/pull/633/commits/d08d04f56d7852a9e1708985586cdb187f385e0b